README.md

```diff
@@ -171,6 +171,111 @@ See the [detailed query manager docs][query-manager-api] for more details on
+- ) because subscriptions are async by nature, you have to listen for these events and act accordingly.
+- ) subscriptions require websockets, so must configure your `link` accordingly
+
+**Creating your subscription:**
+
+`my-app/gql/subscription/new-human`
+
+```
+subscription {
+  newHuman() {
+    name
+  }
+}
+```
+
+**Subscribing from inside a route:**
+
+`app/routes/some-route.js`
+
+```js
+import Route from "@ember/routing/route";
+import { RouteQueryManager } from "ember-apollo-client";
+import query from "my-app/gql/subscription/new-human";
+
+export default Route.extend(RouteQueryManager, {
+  model() {
+    return this.get("apollo")
+               .subscribe({ query }, "human");
+  },
+
+  setupController(controller, model) {
+    model.on("event", event => alert(`${event.name} was just born!`));
+  },
+});
+```
+
+The big advantage of using the RouteQueryManager is that when you navigate away from this route, all subscriptions created will be terminated. That said, if you want to manually unsubscribe (or are not using the RouteQueryManager) `subscription.unsubscribe()` will do the trick.
+
+**Enabling Websockets**
+
+While this library should work w/ any back-end implementation, here's an example with Authenticated [Phoenix](https://github.com/phoenixframework/phoenix) + [Absinthe](https://github.com/absinthe-graphql/absinthe):
+
+`my-app/services/apollo.js`
+```js
+import ApolloService from 'ember-apollo-client/services/apollo';
+import { Socket } from 'phoenix';
+import { inject as service } from "@ember/service";
+import { createAbsintheSocketLink } from '@absinthe/socket-apollo-link';
+import AbsintheSocket from '@absinthe/socket';
+import { computed } from '@ember/object';
+
+
+export default ApolloService.extend({
+  session: service(),
+
+  link: computed(function () {
+    const socket = new Socket("ws://socket-url", {
+      params: { token: this.get('session.token') },
+    });
+    const absintheSocket = AbsintheSocket.create(socket);
+
+    return createAbsintheSocketLink(absintheSocket);
+  }),
+});
+
+```
+Note: This will switch **all** gql communication to use websockets versus `http`. If you want to conditionally use websockets for only subscriptions (a common pattern) this is where [Apollo Link Composition](https://www.apollographql.com/docs/link/composition.html) comes in. Specifically, the `split` function is what we're after (note we are using [apollo-utilities](https://www.npmjs.com/package/apollo-utilities), a helpful `npm` package):
+
+`my-app/services/apollo.js`
+```js
+import { computed } from '@ember/object';
+import ApolloService from 'ember-apollo-client/services/apollo';
+import { split } from 'apollo-link';
+import { getMainDefinition } from 'apollo-utilities';
+import { createAbsintheSocketLink } from '@absinthe/socket-apollo-link';
+import AbsintheSocket from '@absinthe/socket';
+
+export default ApolloService.extend({
+  session      : service(),
+
+link: computed(function () {
```

Comment:
the indentation and whitespace is a bit weird here, can you clean it up for consistency?